appsdk/configurable_test.go

```diff
@@ -305,9 +274,10 @@ func TestBatchByTime(t *testing.T) {
 	}
 
 	params := make(map[string]string)
+	params[Mode] = BatchByTime
 	params[TimeInterval] = "10"
```

Comment:
is this time duration? if so, it will be like 10s?

Reply:
Yes, it is a duration and agree is should be `10s`. Great catch!

Reply:
ok, there is other place like that too, i just skimmed this one out